src/components/tracked-controls.js

```diff
@@ -18,7 +17,13 @@ module.exports.Component = registerComponent('tracked-controls', {
     controller: {default: 0},
     id: {type: 'string', default: ''},
     idPrefix: {type: 'string', default: ''},
-    rotationOffset: {default: 0}
+    rotationOffset: {default: 0},
+    // Arm model parameters, to use when not 6DOF. (pose hasPosition false, no position)
+    headElement: {type: 'selector'},
+    hand: {type: 'string', default: 'right'},
```

Comment:
use case : hand controls with teleporter; two hands if touch or vive, one hand if daydream (or gearvr) 3DOF, and you can enable zero DOF if you really want... to do the one hand, you need to know which hand you should be treating the single 3DOF controller to be, or else the arm model will be rather broken.